spec/fortnox/api/types/account_number_spec.rb

```diff
@@ -13,19 +13,17 @@
     it { is_expected.to be_nil }
   end
 
-  context 'when AccountNumber created with empty string' do
-    include_examples 'raises ConstraintError', ''
-  end
-
   context 'when AccountNumber created with valid number' do
-    include_examples 'equals input', 1234
+    subject { klass['1234'] }
```

Comment:
`1234` should still be a valid number, right? We want to test both string and integer input. So we should probably add a `when AccountNumber created with valid string` case here instead of changing this integer to string.